<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Shader Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="shader_cbox">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Library</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="library_cbox">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Preset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="preset_cbox">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Inputs</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QListWidget" name="inputs_list">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Chan Resolution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="channelRes_cbox">
         <item>
          <property name="text">
           <string>512</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1024</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2048</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4096</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8192</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Bit Depth</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="bitDepth_cbox">
         <item>
          <property name="text">
           <string>8 Bit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16 Bit</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="savePreset_btn">
         <property name="text">
          <string>Save Preset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addChannels_btn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>With </string>
         </property>
         <property name="text">
          <string>Create Channels</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
